cli/tools/jupyter/mod.rs

```diff
@@ -102,6 +102,10 @@ pub async fn kernel(
     )
     .await?;
   worker.setup_repl().await?;
+  worker.execute_script_static(
+    "40_jupyter.js",
+    include_str!("../../js/40_jupyter.js"),
+  )?;
```

Comment:
Maybe this could be moved to `CliMainWorkerFactory::create_custom_worker` and execute it if the subcommand is `Test` of `Jupyter`? Then we don't have to add a flag.

Reply:
Then the same pattern could be used for `40_testing.js` for relevant subcommands.